terraform/deployments/tfc-configuration/variables-integration.tf

```diff
@@ -205,7 +205,10 @@ module "variable-set-rds-integration" {
           log_statement              = { value = "all" }
           deadlock_timeout           = { value = 2500 }
           log_lock_waits             = { value = 1 }
+          rds.logical_replication    = { value = 1, apply_method = "pending-reboot" }
```

Comment:
I'm tentative about these getting applied _before_ Terraform causes the database to get rebooted, but I'm going to trust Terraform to do the right thing.